packages/tradershub/src/flows/Signup/SignupWrapper/SignupWrapper.tsx

```diff
@@ -22,16 +22,15 @@ const SignupWrapper = () => {
     };
 
     const handleSubmit = () => {
-        // will be added later
+        // logic will be added later
+        setIsOpen(false);
     };
 
-    useEffect(() => {
-        ReactModal.setAppElement('#v2_modal_root');
-    }, []);
+    const customStyles = { ...CUSTOM_STYLES, content: { ...CUSTOM_STYLES.content, overflow: 'unset' } };
```

Comment:
@sergei-deriv why we need to `unset` the overflow?

Reply:
@mahdiyeh-deriv it's because without this property dropdown is not visible outside of the modal. Part inside of the red square was invisible (image attached)

<img width="450" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/f9364304-baab-443a-a7c7-1fae7fefa036">